ill-formedness-indicators/variadic-mkvec/README.md

```diff
@@ -0,0 +1,15 @@
+This package is a demonstration of the Ill-Formedness Indicators technique, derived from Csongor Kiss's blog post [_Detecting the undetectable: custom type errors for stuck type families_](https://blog.csongor.co.uk/report-stuck-families/).
```

Comment:
In the blog post, I've changed `Kiss's` to `Kiss'` throughout, as that is what I was taught many years ago. However, you use the same orthography here, so I looked it up and...apparently it's controversial! Your orthography is recommended by a number of US-based style guides (MLA, APA, Chicago), so _maybe_ it's one of those more subtle British vs. American English things.

Your blog post is written in American English. As such, for consistency's sake, would you like me to revert my `Kiss'` changes in my edit? (I realise it's a minor and subtle point, but as an editor, I don't want my edits to misrepresent you.)

Reply:
I don't think it's a British vs American thing. Try pronouncing it out loud, my guess is that you'll pronounce the ‘s’ that you removed (at any rate, when two things are correct, let's avoid, as editors, changing one for the other just because we have a preference. We should favour the author's preference).

Reply:
Yeah, I don't know. The internet is pretty split on this. Despite the fact, that I've never heard someone _say_ “Charles' cat”, they say “Charles's cat”. So I don't understand why they write it that way (in contrast to “my parents' cat”, which is indeed pronounced as written). For the record, I was taught, specifically, as a kid that you skip the s only in _plurals_ ending in ‘s’, but who knows where French English teachers find their grammar rules?